OrbitGl/App.cpp

```diff
@@ -605,7 +605,7 @@ void OrbitApp::ClearCapture() {
   SelectTextBox(nullptr);
 
   // Trigger deallocation of previous sampling related data.
-  auto empty_sampling_profiler = std::make_shared<SamplingProfiler>(Capture::GTargetProcess);
+  auto empty_sampling_profiler = std::make_shared<SamplingProfiler>(std::make_shared<Process>());
```

Comment:
Hi Florian, this breaks sampling, we don't have symbols anymore in the sampling report. We need to pass in the target process to the sampling profiler. I think this is related to the fact that ClearCapture is called after we start a catpure, which is a bit weird.  To repro, load symbols, take a capture, and see that there are no symbols in the sampling report.

Reply:
This seems to fix the issue:
auto empty_sampling_profiler = std::make_shared<SamplingProfiler>(GetSelectedProcess());

Reply:
Okay, that is then super suspicious. I'll fix it with the GetSelectedProcess for now and will have a closer look into why this is the case.

Reply:
It for now uses GetSelectedProcess (which is equivalent to the prior usage of GTargetProcess).
I guess, we should move the assignment of the new SamplingProfiler to StartCapture though.

Reply:
Assigning the new sampling profiler in ClearCapture makes sense as it frees memory when a user presses the clear capture button, I would leave it here.